appveyor.yml

```diff
@@ -41,7 +53,8 @@ install:
   # how to run such command on Windows :)
   # - ps: "ls \"C:/\""
   # - ps: "ls \"C:/Program Files/Microsoft SDKs/Windows\""
-  # - ps: echo $env:PYTHON $env:TOXENV
+  # See the environment variables used for this build.
+  - ps: echo $env:PYTHON $env:TOXENV $env:TWISTED_REACTOR
```

Comment:
Not needed.  The PYTHON version is printed out elsewhere, and we can
already see which reactor is being used in the --reactor argument to trial